### DISCUSSION TITLE
Filter data in XML POST response?

### DISCUSSION THREAD

**Discussion by chuggan** [grafana/grafana-infinity-datasource#302] [Specific APIs] [answered]
Hi All, 

See screenshot - I'm trying to put data from this XML response into a table -- is that even possible? Not sure this specific case is covered in the examples provided, I've sent a POST to the system and data is in the response as per screenshot below. I've been messing about with the rows/root configuration but I'm not quite sure if I'm missing something to get the data I want populated into the panel. Or perhaps it is not supported. Many thanks for any help you can provide.

![image](https://user-images.githubusercontent.com/53254670/164336265-46d98be9-1a2f-4f27-b0ef-aa20cbb92d6c.png)

**Comment by chuggan** [comment, marked as answer]
Hey all, with a fresh mind this morning I edited the rows/roots section and got it working successfully... the data is just nested a bit more than usual. It looked like this: S:Envelope.S:Body[0].ns2:retrieveEquipmentResponse[0].status

All working, very impressive plugin.

**Reply by yesoreyeram** [reply]
Glad it worked for you.. btw, did you try UQL ? That's more powerful and simple. Give a try..
